docker-multinode/common.sh

```diff
@@ -161,7 +154,7 @@ kube::multinode::start_flannel() {
     /opt/bin/flanneld \
       --etcd-endpoints=http://${MASTER_IP}:2379 \
       --ip-masq="${FLANNEL_IPMASQ}" \
-      --iface="${NET_INTERFACE}"
+      --iface="${IP_ADDRESS}"
```

Comment:
Can't we remove this line? The documentation says 

>  Defaults to the interface for the default route on the machine.

That's the same we do. I tested on Ubuntu with "enp0s25" interface and it worked